areas.html

```diff
@@ -3,11 +3,23 @@
 <head>
     <link href="css/grid.css" rel="stylesheet">
     <style>
+        body {
+            margin: 0;
+        }
         .grid {
-            grid-template-areas: "a b b c"
-                                 "d b b c"
-                                 "e e e e";
-            width: 150px;
+            grid-template-columns: repeat(3, 1fr) repeat(2, 2fr);
+            grid-template-rows: repeat(5, 1fr);
+            grid-template-areas: "a b b . ."
+                                 ". b b . c"
+                                 "d b b . c"
+                                 "d . . . c"
+                                 "e e e e e";
+            color: #f00;
+            font-family: sans-serif;
+            font-size: 70px;
+            font-weight: bold;
```

Comment:
I don't care if you change the font sizes, it doesn't seem very relevant though.